src/shared/dissect-image.c

```diff
@@ -1430,26 +1430,29 @@ static int verity_partition(
                  * Improvements in libcrypsetup can ensure this never happens: https://gitlab.com/cryptsetup/cryptsetup/-/merge_requests/96 */
                 if (r == -EINVAL && FLAGS_SET(flags, DISSECT_IMAGE_VERITY_SHARE))
                         return verity_partition(m, v, root_hash, root_hash_size, verity_data, NULL, root_hash_sig ?: hash_sig_from_file, root_hash_sig_size, flags & ~DISSECT_IMAGE_VERITY_SHARE, d);
-                if (!IN_SET(r, 0, -EEXIST, -ENODEV))
+                if (!IN_SET(r, 0, -EEXIST, -EBUSY, -ENODEV))
```

Comment:
any chance you can add some explanatory comments here? i.e. what does EEXIST, EBUSY and ENODEV mean precisely: 

```c
if (!IN_SET(r,
        0,  /* Sucess! */
        -EEXIST, /* Volume is already set up */
        -EBUSY, /* ??? */
        -ENODEV /* ??? */ ) …
```